src/functions/tester/mod.ts

```diff
@@ -0,0 +1,55 @@
+import {
```

Comment:
Moved `SlackFunctionTester` into it's own folder, but this file is generally the same as before — going to call out the big changes so it's easier to review.